.travis.yml

```diff
@@ -0,0 +1,29 @@
+sudo: required
+
+language: ruby
+
+services:
+  - docker
+
+env:
+  - VAGRANT_DEFAULT_PROVIDER=docker
+
+cache:
+  directories:
+    - ~/.berkshelf
+    - ~/.vagrant.d
+
+before_install:
+  - wget https://releases.hashicorp.com/vagrant/2.2.2/vagrant_2.2.2_x86_64.deb
+  - sudo dpkg -i vagrant_2.2.2_x86_64.deb
+
+install:
+  - vagrant up --no-provision
+  - UPDATE_VM_FLAGS=--provision-only vagrant provision
+
+script:
+  - UPDATE_VM_FLAGS=--verify-only vagrant provision
+  
+after_script:
```

Comment:
The whole `after_script` part which was supposed to get the junit style reports out of the VM and store it in `$CIRCLE_TEST_REPORTS` where circleci would find it and render it unfortunately does not work anymore with travisci.

There ain't no such thing as storing test reports or any kind of file with travisci, the only suggestion is to use glorious hacks like these: https://stackoverflow.com/a/40967515

However, since we output the specs in well readable doc format in the console anyway at the end of the build, the html report was just a nice-to-have but never a real necessity

Reply:
Hm... I see...

I probably just removed after script to get it working on travis without giving much thought to it. Having parsable reports would be nice though, could be a reason to look into CircleCI 2.0